### ISSUE TITLE
1.7.1.3 conflict with Toolset/WP-Types

### ISSUE THREAD

**Issue by mattetc** [CalderaWP/Caldera-Forms#2600]
I’m not using Pods or Ultimate Member, but I am still having problems with 1.7.1.3. I tried the .1 and .2 updates yesterday also, and an still getting the same Call to undefined function get_home_path() error.

I am using Toolset/WP-Types, which might be where the conflict resides. They just pushed out a 3.0 release which I also updated yesterday.

Today I have rolled back to Types 2.2.23, which worked fine with Caldera 1.6.3. I then tried to update Caldera from 1.6.3 to 1.7.1.3 and the error occurred again.

Next, on a staging server I deleted all Toolset plugins completely and then updated Caldera to 1.7.1.3 and I did not get the error.

To summarize, even Types 2.x conflicts with Caldera 1.7.1.3. But they have just released 3.0 so the resolution of the conflict will need to work with both.

**Comment by Shelob9** [comment]
This should be fixed in 1.7.1.4 see: https://github.com/CalderaWP/Caldera-Forms/issues/2574#issuecomment-395792791
